easybuild/easyconfigs/p/python-openstack/python-openstackclient-3.12.0-foss-2015b-Python-2.7.11.eb

```diff
@@ -0,0 +1,174 @@
+easyblock = 'Bundle'
+
+name = 'python-openstackclient'
+version = '3.12.0'
+
+homepage = "https://pypi.python.org/pypi/python-openstackclient/"
+description = """OpenStack Client"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+python = 'Python'
+pythonversion = '2.7.11'
+py_short_ver = '.'.join(pythonversion.split('.')[0:2])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    (python, pythonversion),
+    ('libffi', '3.2.1'),    # for pyOpenSSL
+]
+
+#osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

Comment:
Why is this commented out?

Reply:
I used one of the `Python` easyconfigs as a template. Given that `pyOpenSSL` is a dependency, I wasn't sure if this was required or not so left it in. Can you comment @boegel ?

Reply:
`pyOpenSSL` is just a wrapper around `OpenSSL`, so OpenSSL is indeed required, and we should prefer not installing it as a dependency to ensure version updates are picked up from the OS...